apps/metrics/views.py

```diff
@@ -157,9 +184,41 @@ class DeveloperFilter(filters.FilterSet):
         lookup_expr='lte',
         label="Max Application Count")
 
+    first_active_after = filters.DateFilter(
+        label="Date first_active is greater than or equal to",
+        field_name="first_active",
+        lookup_expr='gte')
+    first_active_before = filters.DateFilter(
+        label="Date first_active is less than or equal to",
+        field_name="first_active",
+        lookup_expr='lte')
+
+    last_active_after = filters.DateFilter(
+        label="Date last_active is greater than or equal to",
+        field_name="last_active",
+        lookup_expr='gte')
+    last_active_before = filters.DateFilter(
+        label="Date last_active is less than or equal to",
+        field_name="last_active",
+        lookup_expr='lte')
+
+    active_app_count = filters.NumberFilter(
+        field_name="active_app_count",
+        label="Active Application Count")
+    min_active_app_count = filters.NumberFilter(
```

Comment:
I don't think these filters make sense, this is the serializer for the `Application` model?

Reply:
the way github displayed this confused me, disregard this.